Library/Homebrew/cmd/update.sh

```diff
@@ -242,6 +242,14 @@ EOS
   INITIAL_BRANCH="$(git symbolic-ref --short HEAD 2>/dev/null)"
   if [[ "$INITIAL_BRANCH" != "$UPSTREAM_BRANCH" && -n "$INITIAL_BRANCH" ]]
   then
+
+    if [[ -z "$HOMEBREW_DEVELOPER" ]]
+    then
+      echo "Checking out $UPSTREAM_BRANCH in $DIR..."
+      echo "To checkout $INITIAL_BRANCH in $DIR run:"
+      echo "  'cd $DIR && git checkout $INITIAL_BRANCH"
```

Comment:
To be pedantic, you need to quote `$DIR` and `$INITIAL_BRANCH` for robustness:

```bash
echo "  cd $(printf %q "$DIR") && git checkout $(printf %q "$INITIAL_BRANCH")"
```

But I guess I'm just being pedantic.

P.S. In Zsh the completely robust version (with even nicer output than the Bash version above) would be

```zsh
cd "${(qq)DIR}" && git checkout "${(qq)INITIAL_BRANCH}"
```

This is one of the things I dislike about Bash: wimpy parameter expansion.